rules/prebuilt_framework.bzl

```diff
@@ -0,0 +1,93 @@
+load("@build_bazel_rules_apple//apple:apple.bzl", "apple_dynamic_framework_import", "apple_static_framework_import")
+load("@build_bazel_rules_apple//apple/internal:apple_framework_import.bzl", "AppleFrameworkImportInfo")
+load("@build_bazel_rules_swift//swift/internal:providers.bzl", "SwiftUsageInfo")
+
+def apple_dynamic_framework_import_patched(name, **kwargs):
```

Comment:
should we use the name `apple_dynamic_framework_import`, since it should be a drop-in replacement?

Reply:
Mmmm maybe, only drawback I see is how to use it if you want to mix both this and the existing one

Reply:
Done

Reply:
Ups no. I am getting the error:

```
ERROR: /Users/andres.luque/Git/rules_ios/rules/apple_patched.bzl:7:5: cannot reassign global 'apple_dynamic_framework_import' (read more at https://bazel.build/versions/master/docs/skylark/errors/read-only-variable.html)
ERROR: /Users/andres.luque/Git/rules_ios/rules/apple_patched.bzl:3:51: 'apple_dynamic_framework_import' previously declared here
```

This is because I need to use `apple_dynamic_framework_import` inside the patched `apple_dynamic_framework_import`, and names conflict. Do you know any way to go around this?

Reply:
Ups no. I am getting the error:

```
ERROR: /Users/user/Git/rules_ios/rules/apple_patched.bzl:7:5: cannot reassign global 'apple_dynamic_framework_import' (read more at https://bazel.build/versions/master/docs/skylark/errors/read-only-variable.html)
ERROR: /Users/user/Git/rules_ios/rules/apple_patched.bzl:3:51: 'apple_dynamic_framework_import' previously declared here
```

This is because I need to use `apple_dynamic_framework_import` inside the patched `apple_dynamic_framework_import`, and names conflict. Do you know any way to go around this?

Reply:
`load("foo.bzl", _ apple_dynamic_framework_import = apple_dynamic_framework_import)` lets you re-name a symbol when importing

Reply:
@acecilia I've merged this already to avoid rebase hell but if you want to do these cleanups in a follow up PR you're welcome to.

Reply:
Thanks guys :) will open a new PR with the last fixes shortly